packages/CREATING_PLUGINS.md

```diff
@@ -0,0 +1,68 @@
+# Creating plugins
+
+## Setup
+
+To ensure the plugins are usable by the tool please use the following setup.
+
+### Structure
+
+```
+example-plugin
+- src
```

Comment:
Our example plugin uses lib instead of src. We should pick one and be consistent. I think the lib makes more sense

Reply:
fixed